code/game/objects/items/crayons.dm

```diff
@@ -803,6 +803,8 @@
 		/obj/item/fish, // Used for aquarium sprites
 		/obj/structure/window, // Does not play nice with window tint
 	))
+	// The Spinward Stellar Coalition's D.A.R.E. campaign was effective in persuading NT to stop selling spraypaint cans to people who can't be trusted to not get high off of it.
+	age_restricted = TRUE
```

Comment:
spraypaint isn't actually sold anywhere though? Do we display this in-game anywhere, if not then this is pointless. Twice is pointless unless you add paint sniffing.

Reply:
We sell it in the arts and crafts vendor, I thought. You get high if you're spraypainted; spraypaint is welding fuel and ethanol and it reagent transfers it to people you spray in the face.